### ISSUE TITLE
Error with QGIS 3.10.1

### ISSUE THREAD

**Issue by dfelikson** [minorua/Qgis2threejs#225]
I installed Qgis2threejs via the plugin manager, started a new project and imported a DEM. When I click the Qgis2threejs Exporter button, I get the following error:

```
2020-08-28T13:53:18     INFO    Opening Qgis2threejs Exporter...
2020-08-28T13:53:18     WARNING    Traceback (most recent call last):
              File "/Users/dfelikso/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/q3dcontroller.py", line 322, in _processRequests
              self.buildScene()
              File "/Users/dfelikso/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/q3dcontroller.py", line 194, in buildScene
              self.iface.loadJSONObject(self.builder.buildScene(False))
              File "/Users/dfelikso/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/build.py", line 56, in buildScene
              "proj": crs.toProj4() if Qgis.QGIS_VERSION_INT              AttributeError: 'QgsCoordinateReferenceSystem' object has no attribute 'toProj'
             
2020-08-28T13:53:35     WARNING    Qgis2threejs Exporter has closed.
```

**Comment by Camischflr** [comment]
Hi @dfelikson, I also have this problem using QGIS version 3.10.1 and Qgis2threejs version 2.5 ! Did you find a solution? Regards

**Comment by minorua** [comment]
Completely fixed in version 2.6. Thanks!
